EAD/Session 1/Assignment1/index.html

```diff
@@ -0,0 +1,505 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+    <meta charset = "UTF-8">
+    <title>Document</title>
+    <style>
+        body
+        {
+            background-color : rgb(143, 174, 188);
+        }
+        h1
+        {
+            margin-top : 30px;
+        }
+        td
+        {
+            width : 200px;
+            padding-top : 10px;
+        }
+        input
+        {
+            border-style : groove;
+            border-radius : 30px;
+            padding : 5px;            
+        }
+        textarea,select
+        {
+            border-radius : 10px;
+        }
+        hr
+        {
+            width : 100%;
+            height : 2px;
+            color : black;
+        }
+        #costField
+        {
+            padding-left : 23px;
+        }
+    </style>
+
+    <script>

```

Comment:
We can add javascript in another file and the js file in our file

Reply:
Ok, I will next time.